pod.go

```diff
@@ -144,4 +158,16 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 		addGauge(descPodContainerStatusReady, boolFloat64(cs.Ready), cs.Name)
 		addCounter(descPodContainerStatusRestarts, float64(cs.RestartCount), cs.Name)
 	}
+
+	nodeName := p.Spec.NodeName
```

Comment:
Hi Frederic, thanks for the detailed explanation. I am not very familiar with the sequence NodeName goes through at this point :(